patches/node/enable_crashpad_linux_node_processes.patch

```diff
@@ -0,0 +1,60 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: VerteDinde <keeleymhammond@gmail.com>
+Date: Sun, 20 Nov 2022 21:45:20 -0800
+Subject: fix: enable crashpad for ELECTRON_RUN_AS_NODE linux processes
+
+Passes the crashpad handler PID and crashdump signal file descriptor
+to child processes spawned with `ELECTRON_RUN_AS_NODE` which is used
+by the crashpad client to connect with the handler process.
+
+diff --git a/lib/child_process.js b/lib/child_process.js
+index 73c1dc769542865bdf7a2a03c16cef141d3f4b05..4c725a991b803c2bfde58b674df15b328102450b 100644
+--- a/lib/child_process.js
++++ b/lib/child_process.js
+@@ -159,7 +159,6 @@ function fork(modulePath, args = [], options) {
+       ArrayPrototypeSplice(execArgv, index - 1, 2);
+     }
+   }
+-
+   args = [...execArgv, modulePath, ...args];
+ 
+   if (typeof options.stdio === 'string') {
+@@ -613,6 +612,22 @@ function normalizeSpawnArguments(file, args, options) {
+                     'options.windowsVerbatimArguments');
+   }
+ 
++  if (process.platform === 'linux') {
++    if (ObjectPrototypeHasOwnProperty(options.env || {}, 'ELECTRON_RUN_AS_NODE') &&
++        (file === process.execPath)) {
++      // On Linux, pass the file descriptor which crashpad server process
++      // uses to monitor the child process.
++      const fd = process.getCrashdumpSignalFD();
++      if (fd !== -1) {
++        options.env.CRASHDUMP_SIGNAL_FD = fd;
++        // On Linux, pass the PID of the crashpad handler process to the child process
++        // which is needed by the crashpad client running in the child process.
++        // https://source.chromium.org/chromium/chromium/src/+/110.0.5415.0:components/crash/core/app/crashpad_linux.cc;l=199-206
++        ArrayPrototypePush(args, `--crashpad-handler-pid=${process.getCrashpadHandlerPID()}`);
```

Comment:
I think we can't really do this, this changes the process.argv in the child process, which will break apps. Perhaps we can also send this via environment variable, and then push it into base::CommandLine in the child process? Or, alternatively, _remove_ it from the command line in the child process before Node.js gets to it (is that possible?)

Reply:
Yup this is being handled in `node_main`, continuing the comments in https://github.com/electron/electron/pull/36460#discussion_r1033935063